build.gradle

```diff
@@ -49,8 +49,8 @@ dependencies {
     compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.1'
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.3.2'
     compile group: 'org.jsoup', name: 'jsoup', version: '1.9.2'
-    compile group: 'com.ibm.watson.developer_cloud', name: 'conversation', version:'3.5.3'
-    compile group: 'com.ibm.watson.developer_cloud', name: 'discovery', version:'3.5.3'
+    compile group: 'com.ibm.watson.developer_cloud', name: 'conversation', version:'3.8.0'
+    compile group: 'com.ibm.watson.developer_cloud', name: 'discovery', version:'3.8.0'
```

Comment:
//@Test
public void testSendMessage() throws IOException, InterruptedException {
       Test is broken with 3.8.0 update, and I needed to get a fix out quickly, will fix very soon

Use the constant from the SDK
      It does, just keeps it in one place Constants.java